src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -476,6 +476,21 @@ def validate_max_surge(namespace):
         raise CLIError("--max-surge should be an int or percentage")
 
 
+def validate_max_unavailable(namespace):
+    """validates parameters max unavailable are positive integers or percents."""
+    if namespace.max_unavailable is None:
+        return
+    int_or_percent = namespace.max_unavailable
+    if int_or_percent.endswith('%'):
+        int_or_percent = int_or_percent.rstrip('%')
+
+    try:
+        if int(int_or_percent) < 0:
+            raise CLIError("--max-unavailable must be positive")
```

Comment:
Please use the specific error type `InvalidArgumentValueError` instead of `CLIError`

Reply:
updated, PTASL, thank you
@zhoxing-ms